src/index.js

```diff
@@ -43,6 +69,44 @@ async function publishToSQS(data) {
   }
 }
 
+/**
+ * send IndexerNotified event to the appropriate SNS topic
+ * @see https://github.com/elastic-ipfs/metrics-collector/blob/main/src/indexer-events/examples/IndexerNotified.json
+ * @param {object} arg.s3 - s3 object info
+ * @param {string} arg.s3.bucket.name - s3 bucket name
+ * @param {string} arg.s3.object.key - s3 key name
+ * @param {number} arg.s3.object.size - object size in bytes
+ * @param {Date} [arg.startTime] - time of IndexerNotified event
+ */
+function createIndexerNotifiedEvent({ s3, startTime = new Date() }) {
+  const event = {
```

Comment:
You can directly return here.

Reply:
https://github.com/elastic-ipfs/bucket-to-indexer/pull/13/commits/5e9acf65256271c3f2a8dbe2272de22b8f3f4a8e